fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -24,6 +32,15 @@ def self.start
     ensure
       FastlaneCore::UpdateChecker.show_update_status('fastlane', Fastlane::VERSION)
       Fastlane::PluginUpdateManager.show_update_status
+      if $trace
+        $trace_output = $stdout.string.gsub(/\e\[([;\d]+)?m/, '')
```

Comment:
This looks like it removing ANSI color escape codes, but that's definitely worth documenting in a comment, or putting in a named method to make it clearer 👍